### PR TITLE
Introduce and Use NetworkIndicator component

### PR DESCRIPTION
**Pull request by jarolrod**
Signals to the user when they are running on a network other than main.

Closes https://github.com/bitcoin-core/gui-qml/issues/233

#### Light
| testnet | signet | regtest |
| ------- | ------ | ------- |
| Screen Shot 2023-02-08 at 6 48 47 PM |  Screen Shot 2023-02-08 at 6 39 00 PM | Screen Shot 2023-02-08 at 6 39 21 PM |

#### Dark
| testnet | signet | regtest |
| ------- | ------ | ------- |
| Screen Shot 2023-02-07 at 5 24 23 PM | Screen Shot 2023-02-07 at 5 24 50 PM | Screen Shot 2023-02-07 at 5 25 34 PM |

[Windows](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/win64/insecure_win_gui.zip?branch=pull/<PR>)
[Intel macOS](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/macos/insecure_mac_gui.zip?branch=pull/<PR>)
[Apple Silicon macOS](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/macos_arm64/insecure_mac_arm64_gui.zip?branch=pull/<PR>)
[ARM64 Android](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/android/insecure_android_apk.zip?branch=pull/<PR>)

### CODE REVIEW

**Comment by jarolrod** [comment]
Updated from [bcb3aee ](https://github.com/bitcoin-core/gui-qml/commit/bcb3aeefeb89f58b279776a1cb5803b547152624)to [dd76b6e](https://github.com/bitcoin-core/gui-qml/commit/dd76b6ecced71cf3b48adf76265769258277b5e0), [compare](https://github.com/bitcoin-core/gui-qml/compare/bcb3aeefeb89f58b279776a1cb5803b547152624..dd76b6ecced71cf3b48adf76265769258277b5e0)

- Changes: I've been informed by @GBKS that the text should always be white

**Comment by hebasto** [comment]
Rebase?

**Comment by johnny9** [comment]
ACK 932622e674b6e2476f635e901b2b00cbc58b7e6a

Changes look good and match figma spec.

Screenshot from 2023-02-08 22-17-32
Screenshot from 2023-02-08 22-17-10

**Review by johnny9 — APPROVED** [review]
ACK 932622e674b6e2476f635e901b2b00cbc58b7e6a

Changes look good, matches the figma spec, and no errors.

Screenshot from 2023-02-08 22-17-32
Screenshot from 2023-02-08 22-17-10

File: src/qml/pages/node/NodeRunner.qml [hunk 1]
```diff
@@ -18,7 +18,15 @@ Page {
 
     Component.onCompleted: nodeModel.startNodeInitializionThread();
 
-    BlockClock {
+    ColumnLayout {
+        spacing: 30
         anchors.centerIn: parent
+        BlockClock {
+          id: blockClock
```
Comment (johnny9) [inline src/qml/pages/node/NodeRunner.qml:25]: Whitespace issue here

**Review by johnny9 — CHANGES_REQUESTED** [review]
minor whitespace issue.

File: src/qml/pages/node/NodeRunner.qml (hunk 1 above)
Comment (jarolrod) [reply src/qml/pages/node/NodeRunner.qml]: thanks, fixed

**Comment by jarolrod** [comment]
Updated from [932622e ](https://github.com/bitcoin-core/gui-qml/commit/932622e674b6e2476f635e901b2b00cbc58b7e6a)to [01b3909](https://github.com/bitcoin-core/gui-qml/commit/01b3909f611b3a1db4d1d827350ab2b72c97e2f6), [compare](https://github.com/bitcoin-core/gui-qml/compare/932622e674b6e2476f635e901b2b00cbc58b7e6a..01b3909f611b3a1db4d1d827350ab2b72c97e2f6)

changes: addressed indenting issue but also removed the id for the block clock component since its not being used here.

**Review by johnny9 — APPROVED** [review]
ACK 01b3909f611b3a1db4d1d827350ab2b72c97e2f6

Changes look good, no errors, and matches the figma

Screenshot from 2023-02-09 11-53-42
Screenshot from 2023-02-09 11-54-05

**Comment by GBKS** [comment]
Sorry for the late feedback. A couple of details don't match the design. The font should be 15px, and vertical spacing should be 2 (horizontal remains 7). As is right now, the indicator looks slightly too big compared to everything else.

**Pull request merged**